defaults/main.yml

```diff
@@ -9,6 +9,12 @@ jenkins_version: 2.52 # latest
 
 jenkins_volume: "/dev/xvdb"
 
+jenkins_use_static_ssh_key: false
+
+jenkins_user_private_ssh_key: ""
+
+jenkins_user_public_ssh_key: ""
```

Comment:
so...this will be populated only after a person has properly configured vault...and yeah..it will be environment based...the lookup plugin that vishesh has written for vault uses grofers_environment to decide vault address....
if its not able to read, then it will raise an error..
and also the check will be valid only when a person has enabled static key using jenkins_use_static_ssh_key ...so if empty or not able to read, then it will fail...
so no default value...since it should fail in case nothing is provided after enabling